### DISCUSSION TITLE
GetInboundUsersCountResponse and get_user_count methods are available but not present in HandlerService

### DISCUSSION THREAD

**Discussion by the2pizza** [XTLS/Xray-core#4149] [General]
Hi folks,

I see we have methods in proto files to retrieve user info and user_count for a inbound, but my Hadler service doesn't expose those methods, are there a bug or I should implement server side on my own? 

```
grpcurl -plaintext 127.0.0.1:23456 list xray.app.proxyman.command.HandlerService
xray.app.proxyman.command.HandlerService.AddInbound
xray.app.proxyman.command.HandlerService.AddOutbound
xray.app.proxyman.command.HandlerService.AlterInbound
xray.app.proxyman.command.HandlerService.AlterOutbound
xray.app.proxyman.command.HandlerService.RemoveInbound
xray.app.proxyman.command.HandlerService.RemoveOutbound
```

```
message GetInboundUserRequest {
  string tag = 1;
  string email = 2;
}

message GetInboundUserResponse {
  repeated xray.common.protocol.User users = 1;
}

message GetInboundUsersCountResponse {
  int64 count = 1;
}

```

**Comment by yuhan6665** [comment]
You might have an older version of xray. You can try these commands as well

./xray api inboundusercount --server=127.0.0.1:10085 -tag="vmess"
./xray api inbounduser --server=127.0.0.1:10085 -tag="vless" -email="vless@example.com"

**Reply by the2pizza** [reply]
Oh thanks, could you suggest which verison Xray-core is it?

**Comment by the2pizza** [comment]
Thanks @yuhan6665 I downloaded latest release and it works, much appreciate
